core/trino-spi/src/main/java/io/trino/spi/block/AbstractRowBlock.java

```diff
@@ -38,6 +38,7 @@
 
```

Comment:
Improve commit message:
```
When there are no null rows within RowBlock
then it's not needed to store fieldBlockOffsets.
This improves performance and reduces
network traffic as fieldBlockOffsets
array is not serialized or deserialized.
```

Also, attach before/after benchmark results to this commit message